### ISSUE TITLE
The equation of the R_spec calculation

### ISSUE THREAD

**Issue by harmomyshoes** [Eomys/MoSQITo#57]
The code on the roughess_dw_main_calc function underneath 

>     R_spec[0] = gzi[0] * pow(mod_depth[0] * ki[0], 2)
>     R_spec[1] = gzi[1] * pow(mod_depth[1] * ki[1], 2)
>     for i in np.arange(2, 45):
>         R_spec[i] = gzi[i] * pow(mod_depth[i] * ki[i] * ki[i - 2], 2)
>     R_spec[45] = gzi[45] * pow(mod_depth[45] * ki[43], 2)
>     R_spec[46] = gzi[46] * pow(mod_depth[46] * ki[44], 2) 

then I check the "Handbook of Signal Processing in Acoustics"
 by Peter Daniel. 
![image](https://user-images.githubusercontent.com/12989339/198319935-f56bd9c1-e86e-4ad6-b36f-b531538ea8d1.png)
in the book, the gzi seems to need a pow^2, 

whether I have some misunderstandings here.

**Comment by wantysal** [comment]
Hi, sorry for the late reply...

There is indeed a gray area in this part of the calculation since the article by P. Daniel does not provide the coefficients g(zi)...
We used the Aures coefficients and then tried to fit the model to the results given by Daniel and Weber. It appeared that the g(zi) coefficients we used were already squared since they provide better results.

Thank you for noticing this, if you have more data about this calculation, feel free to contribute to MOSQITO development ;)
Salomé
